lamindb/core/storage/_backed_access.py

```diff
@@ -208,7 +208,9 @@ def safer_read_partial(elem, indices):
                 ds = CSRDataset(elem)
                 result = _subset_sparse(ds, indices)
             except Exception:
-                pass
+                logger.debug(
+                    "Encountered an exception while attempting to subset a sparse dataset by indices."
```

Comment:
If we log something, here, we should also log `e` where `e` is the raised instance of `Exception`.